### PR TITLE
e2e tests (initial setup)

### PR DESCRIPTION
**Pull request by hamidra**
This provides a workflow and playwright setup to run e2e tests using:
playwright + zombienet 
and includes a simple connection test to verify app state transition to 'ConnectedState'

### CODE REVIEW

File: .github/workflows/playwright-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: |
+          npm install -g yarn
+          yarn install
+
+      # Spawn a zomboienet chain in background
+      # ToDo: switch to zombienet podman when static ports are supported
+      # podman is preferred but currently it is not possible to set static ports
```
Comment (jeluard) [inline .github/workflows/playwright-e2e.yml:23]: @hamidra Do we have a GH issue tracking this?

File: package.json [hunk 2]
```diff
@@ -35,6 +37,7 @@
     "@polkadot/react-identicon": "2.10.1",
     "@react-spring/web": "9.6.1",
     "@use-gesture/react": "10.2.23",
+    "commander": "9.5.0",
```
Comment (jeluard) [inline package.json:40]: Do we need that?

File: .github/workflows/playwright-e2e.yml (hunk 1 above)
Comment (hamidra) [reply .github/workflows/playwright-e2e.yml:23]: Good point.
I have an issue open in zombienet repo https://github.com/paritytech/zombienet/issues/704
I'll open an issue here to track as well.

Comment (hamidra) [reply .github/workflows/playwright-e2e.yml:23]: https://github.com/jeluard/swipealot/issues/115

File: package.json (hunk 2 above)
Comment (hamidra) [reply package.json:40]: This is needed till the below fix is merged.
https://github.com/paritytech/zombienet/pull/703

Comment (jeluard) [reply package.json:40]: Now fixed upstream

**Pull request merged**